### DISCUSSION TITLE
comment color?

### DISCUSSION THREAD

**Discussion by umop3plsdn** [trizen/pipe-viewer#83] [Ideas]
Not sure how hard it would be but would it be possible to have the option to colorize at least the headers like the username of the comment? Even though it is bolded it is still a little difficult to read it blends together

**Comment by benoit-pierre** [comment]
#123 comes with some [settings to change the colors of the comments](https://github.com/trizen/pipe-viewer/blob/main/bin/gtk-pipe-viewer#L231).

**Reply by umop3plsdn** [reply]
I noticed that's for the gtk-viewer does that work also in the CLI?
